app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,64 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def channels
+
+    if params[:uuid].nil?
+
+      channels = Channel.all.map { |channel| channel.id }
+      data = JSON.pretty_generate(channels)
```

Comment:
`pretty_generate` is not what you're looking for when you sample the response. I'd recommend `channels.to_json`.

Reply:
Fixed